.taskcat.yml

```diff
@@ -24,6 +24,7 @@ tests:
       pSRAStagingS3KeyPrefix: $[taskcat_project_name]
       pEnableCloudTrial: 'true'
```

Comment:
Variable was fixed to `pEnableCloudTrail` elsewhere but still `pEnableCloudTrial` here.

Reply:
fixed.